lesson_2/test5.py

```diff
@@ -13,13 +13,9 @@ def merge(list1, list2):
     new_list = list(zip_list)
     # SET merge_list = to append values of tuple
     merge_list = []
-    # SET tuple1, tuple2 to store values from in tuple
-    tuple1 = 0
-    tuple2 = 0
+    
     # WHILE for items in new_list index items from tuple and append to merge_list
-    for element in new_list:
-        tuple1 = element[0]
-        tuple2 = element[1]
+    for tuple1, tuple2 in new_list:
```

Comment:
This is good but the variable names are not right -- we are not dealing with multiple tuples per iteration; instead we are dealing with a single tuple per iteration that has multiple elements.

I would write this as `for firstElement, secondElement in new_list:`

Let me know if that makes sense.

Reply:
YUH thought about it when I made it, agreed it could def be named better